src/radical/pilot/agent/scheduler/base.py

```diff
@@ -903,82 +907,12 @@ def _try_allocation(self, task):
         self._change_slot_states(slots, rpc.BUSY)
         task['slots'] = slots
 
-        if slots['lfs_per_node']['path']:
-            task['description']['environment']['NODE_LFS_PATH'] = \
-                slots['lfs_per_node']['path']
```

Comment:
Should this part be kept? or it was moved?

Reply:
Ack, readded (changed the name for consistency, always set it)

Reply:
is the name `NODE_LFS_PATH` actually used by the machine and not by RP? If it is so then it shouldn't be renamed

Reply:
btw, `Popen` also sets `RP_LFS` ([link](https://github.com/radical-cybertools/radical.pilot/blob/f333d086aeeef51cf4f34c707ceb0aabc4514acb/src/radical/pilot/agent/executing/popen.py#L661)) is it related to this as well? (also don't see where it is used by RP)